apps/admin-x-activitypub/src/components/inbox/FollowSiteModal.tsx

```diff
@@ -1,28 +1,10 @@
+import {useState} from 'react';
+
 import NiceModal from '@ebay/nice-modal-react';
-import {ActivityPubAPI} from '../../api/activitypub';
 import {Modal, TextField, showToast} from '@tryghost/admin-x-design-system';
-import {useBrowseSite} from '@tryghost/admin-x-framework/api/site';
-import {useMutation} from '@tanstack/react-query';
 import {useRouting} from '@tryghost/admin-x-framework/routing';
-import {useState} from 'react';
 
-function useFollow(handle: string, onSuccess: () => void, onError: () => void) {
```

Comment:
Moved this out to `hooks/useActivityPubQueries.ts` so it can also be utilised by `components/global/FollowButton.tsx`